src/Models/Archive.php

```diff
@@ -31,7 +31,7 @@ public function getDataAttribute(string $value): object
      */
     public function getArchivedData(): array
     {
-        return json_decode($this->getRawOriginal('data'), true);
+        return json_decode($this->getOriginal('data'), true);
```

Comment:
I'm a little concerned about this one. I originally had `getOriginal`, but it was returning the json_decoded data from line 24, whereas I was relying on getting the actual string value here. Maybe its a L6 vs L7 thing, or maybe I was just mistaken, just something to keep an eye on though.